libkbfs/dirty_bcache.go

```diff
@@ -34,20 +34,68 @@ type dirtyReq struct {
 //
 // DirtyBlockCacheStandard controls how fast uses can write into KBFS,
 // and does so with a TCP-like slow-start algorithm that adjusts
-// itself according to how fast bytes are synced to the server.  It
-// keeps track of a "sync buffer" that holds all of the bytes that
-// have been written but not yet synced to the server.  When the
-// buffer is full, write requests get blocked by a fraction of the
-// write timeout, according to how close the user is to filling up
-// their "dirty buffer" -- i.e., the total number of bytes for which
-// an overall Sync operation has not yet completed (even if their
-// individual block Puts to the server have completed).
+// itself according to how fast bytes are synced to the server.
+// Conceptually, there are two buffers:
 //
-// The size of the sync buffer can vary between a minimum and maximum.
-// It starts out at the minimum, and grows exponentially as Puts
-// succeed.  However, if a write request is blocked for too long by
-// the backpressure and buffer-fullness checks, the sync buffer size
-// is cut in half.
+//   syncBuf: The bytes that are currently syncing, or have finished
+//   syncing, back to the servers.  Each TLF has only one sync at a
+//   time, but multiple TLFs may be syncing at the same time.  We also
+//   track how many bytes within this buffer have finished syncing.
```

Comment:
How does this affect our tuning for parallel block puts? Do all currently-syncing TLFs get put in the same block put worker queue?